golang/cosmos/daemon/cmd/genaccounts.go

```diff
@@ -70,7 +70,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 				addr = info.GetAddress()
 			}
 
-			coins, err := sdk.ParseCoins(args[1])
+			coins, err := sdk.ParseCoinsNormalized(args[1])
```

Comment:
Did `ParseCoins` just get renamed?

Or is this a semantic change? If so, are any of our tests sensitive enough to fail if this change weren't made?

Reply:
This was a merge from upstream gaiad, and yes, it was renamed (would fail with compile error).

Reply:
Is that whole commit from upstream?

Reply:
Yes, except for a few changes in golang/cosmos/x to fix compilation errors.